azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -388,4 +388,10 @@ public Executor fetchExecutorByExecutionId(int execId) throws ExecutorManagerExc
   public void unassignExecutor(int executionId) throws ExecutorManagerException {
     executionExecutorMapping.remove(executionId);
   }
+
+  @Override
+  public List<ExecutableFlow> fetchRecentlyFinishedFlows(long lifeTimeMs)
+      throws ExecutorManagerException {
+    return null;
```

Comment:
I agree. It's better to use Mockito to mock the executor loader. Currently this class is used in many test cases, but we can gradually retire it in the future.